Make sure the auto-regenerated headers (after scripts/MakeHeader.py) actually compile.

--- a/linux/LinuxProcess.c	2018-04-26 18:17:31.349248819 +0200
+++ b/linux/LinuxProcess.c	2018-04-11 02:58:46.994360617 +0200
@@ -143,7 +143,7 @@
 } LinuxProcess;
 
 #ifndef Process_isKernelThread
-#define Process_isKernelThread(_process) ((LinuxProcess*)(_process)->isKernelThread)
+#define Process_isKernelThread(_process) (((LinuxProcess*)(_process))->isKernelThread)
 #endif
 
 #ifndef Process_isUserlandThread
